src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -329,6 +329,10 @@ If this file comes from a NuGet package, update to a newer version of the NuGet
 The capitalized word "Portable" that appears earlier in the message is plain text and should be translated, but the lowercase word "portable" later in the message is a literal value and should not be translated.
 {0} - The file name of a deprecated symbol file</comment>
   </data>
+  <data name="XA0138" xml:space="preserve">
+    <value>@(AndroidAsset) build action does not support 'AssetPack' or 'DeliveryType' Metadata in Library Projects.</value>
```

Comment:
Might be better to be "differently more direct"?

> %(AndroidAsset.AssetPack) and %(AndroidAsset.AssetPack) item metadata are only supported in Application projects.

or "…when `$(AndroidApplication)`=true"?